sdks/python/apache_beam/io/gcp/pubsub_integration_test.py

```diff
@@ -210,6 +210,8 @@ def test_streaming_data_only(self):
 
   @pytest.mark.it_postcommit
   def test_streaming_with_attributes(self):
+    if self.runner_name == 'TestDataflowRunner':
+      pytest.skip("Sickbay this test due to Dataflow service issues (BEAM-13218)")
```

Comment:
nit: let's just mention the jira for now, it's not clear if this is a service issue.

```suggestion
      pytest.skip("BEAM-13218")
```

This should also help lint/formatter checks, failing for line-too-long